docs/source/getting-started/authenticators-users-basics.md

```diff
@@ -107,8 +107,8 @@ with any provider, is also available.
 
 ## Use DummyAuthenticator for testing
 
-The :class:`~jupyterhub.auth.DummyAuthenticator` is a simple authenticator that
-allows for any username/password unless if a global password has been set. If
+The `DummyAuthenticator` is a simple authenticator that
+allows for any username/password unless a global password has been set. If
```

Comment:
This is an improvement from the current state and we can improve it further at a later point if we want to make this a reference to documentation about the class or similar I think.

![image](https://user-images.githubusercontent.com/3837114/117039347-f499dd00-ad08-11eb-9b0f-f53fb48bbb4c.png)